packages/lit-dev-content/samples/examples/observers-intersection-controller/intersection-demo.ts

```diff
@@ -0,0 +1,71 @@
+import {LitElement, html} from 'lit';
+import {customElement} from 'lit/decorators.js';
+import {map} from 'lit/directives/map.js';
+import {styleMap} from 'lit/directives/style-map.js';
+import {IntersectionController} from '@lit-labs/observers/intersection_controller.js';
+
+import {SHAPES} from './constants.js';
+import {styles} from './styles.js';
+
+/**
+ * Show how a @lit-labs/observers IntersectionController can manage multiple
+ * child elements. This component generates multiple elements, and then toggles
+ * a class as they come in and out of the viewport.
+ */
+@customElement('intersection-demo')
+export class MyElement extends LitElement {
+  private intersectionController!: IntersectionController;
+  static styles = styles;
+
+  render() {
+    return html`
+      <div id="parallax">
+        ${map(
+          SHAPES,
+          (item) =>
+            html`<div
+              class="hidden ${item.class}"
+              style=${styleMap(item.styles)}
+            ></div>`
+        )}
+      </div>
+      <div id="top-line"></div>
+      <div id="bottom-line"></div>
+    `;
+  }
+
+  firstUpdated() {
+    this.intersectionController = new IntersectionController(this, {
```

Comment:
Seems like you can follow the normal convention and make this as a class field.

Reply:
Great point! Done!